### ISSUE TITLE
Project's unarchive error

### ISSUE THREAD

**Issue by marekmrowca** [python-gitlab/python-gitlab#305]
Hi,
I have the project with id #id, which is archived and I want to unarchive it via the API.
Here is the piece of code:
```
gl = gitlab.Gitlab.from_config('gitlab', ['/etc/python-gitlab.cfg'])
projects = gl.projects.get(#id)
projects.unarchive()
```

It returns:

> Traceback (most recent call last):
  File "getGitlabProjects.py", line 9, in <module>
    projects.unarchive()
  File "/usr/lib/python2.7/site-packages/gitlab/v4/objects.py", line 2027, in unarchive
    raise_error_from_response(r, GitlabCreateError, 201)
  File "/usr/lib/python2.7/site-packages/gitlab/exceptions.py", line 204, in raise_error_from_response
    response_body=response.content)
gitlab.exceptions.GitlabCreateError: 404: {"error":"404 Not Found"}

When I archive it with:
`project.archive`
it succeeds
I also tried with api v3 and it didn't work either

Both methods in /usr/lib/python2.7/site-packages/gitlab/v4/objects.py are:
```   
 def archive(self, **kwargs):
        """Archive a project.
        Returns:
            Project: the updated Project

        Raises:
            GitlabCreateError: If the action cannot be done
            GitlabConnectionError: If the server cannot be reached.
        """
        url = "/projects/%s/archive" % self.id
        r = self.gitlab._raw_post(url, **kwargs)
        raise_error_from_response(r, GitlabCreateError, 201)
        return Project(self.gitlab, r.json()) if r.status_code == 201 else self

    def unarchive(self, **kwargs):
        """Unarchive a project.

        Returns:
            Project: the updated Project

        Raises:
            GitlabDeleteError: If the action cannot be done
            GitlabConnectionError: If the server cannot be reached.
        """
        url = "/projects/%s/unarchive" % self.id
        r = self.gitlab._raw_delete(url, **kwargs)
        raise_error_from_response(r, GitlabCreateError, 201)
        return Project(self.gitlab, r.json()) if r.status_code == 201 else self
```

GitLab 9.5.1
GitLab Shell 5.8.0
GitLab Workhorse v3.0.0
GitLab API v4
Git 2.13.5
Ruby 2.3.3p222
Rails 4.2.8
postgresql 9.6.3 

Please help,
Best regards,
Marek

------
In page: api/gitlab.rst

**Comment by gpocentek** [comment]
Thanks for your report.

I believe this is fixed in python-gitlab 1.0.0, released a few hours ago. Could you check with this version?

Thanks.

**Comment by marekmrowca** [comment]
Hi,
I have just checked with version 1.0.0 and it works!
Thank you
